source/server/backtrace.h

```diff
@@ -92,7 +98,14 @@ class BackwardsTrace : Logger::Loggable<Logger::Id::backtrace> {
         obj_name = trace.object_filename;
         ENVOY_LOG(critical, "thr<{}> obj<{}>", thread_id, obj_name);
       }
+
+#ifdef __APPLE__
+      // In the absence of stack_decode.py, print the function name.
+      ENVOY_LOG(critical, "thr<{}> #{} {}: {}", thread_id, stack_trace_[i].idx,
```

Comment:
@dnoe should we do this in non-Apple as well, for those folks who are staring at traces without the benefit of stack_decode.py (e.g. scraped out of a CI system)?